### DISCUSSION TITLE
Array field is defined as String in GraphQL Schema

### DISCUSSION THREAD

**Discussion by prasanna-rkumar** [strapi/strapi#7864] [Legacy Topics]
I have a content type of Teacher. Added a new array field called handling_classes directly in the **teacher.settings.json** file.
`"handling_classes": {
      "type": "array"
    }`
 I then installed GraphQL using `strapi install graphql`. 
But the generated graphql schema has type **String** for **handling_classes** field `handling_classes: String`. 
What am I doing wrong here?

> strapi - 3.1.0-beta.0

**Comment by derrickmehaffy** [comment]
We do not have a model type as `array` I'm curious where you got this from?

**Reply by prasanna-rkumar** [reply]
> We do not have a model type as `array` I'm curious where you got this from?

https://github.com/strapi/strapi/issues/784#issuecomment-401605647 I just saw this and assumed that adding this model type would make wonders. I switched to `JSON` and added values like `["BC", "LP"]`. Hope it suffices my needs.
